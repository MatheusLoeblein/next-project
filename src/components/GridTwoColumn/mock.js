export default {
  title: 'Grid Two Column',
  text: `Lorem ipsum, dolor sit amet consectetur adipisicing elit.
        Voluptatum aut, eaque velit ullam exercitationem architecto
        excepturi asperiores quos cupiditate, explicabo, minima impedit
        distinctio? Officia dicta, possimus modi recusandae explicabo magni!`,
  srcImg: 'assets/images/javascript.svg',
  background: false,
};
