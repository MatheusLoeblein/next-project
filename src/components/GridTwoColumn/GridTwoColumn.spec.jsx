import { renderTheme } from '../../styles/render-theme';
import { GridTwoColumn } from '.';
import { screen } from '@testing-library/react';

import mock from './mock';

describe('<GridTwoColumn />', () => {
  it('should render with two column grid', () => {
    const { container } = renderTheme(<GridTwoColumn {...mock} />);

    expect(container).toMatchSnapshot();
  });
  it('should render with two column grid not background', () => {
    const { container } = renderTheme(
      <GridTwoColumn {...mock} background={undefined} />,
    );

    expect(container).toMatchSnapshot();
  });
});
