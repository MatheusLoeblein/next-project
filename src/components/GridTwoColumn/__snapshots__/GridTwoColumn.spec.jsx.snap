// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridTwoColumn /> should render with two column grid 1`] = `
<div>
  <div
    class="styles__Container-d23vz7-0 bMuKEd"
    id=""
  >
    <div
      class="styles__Container-sc-1abwwhx-0 kXNRnk"
    >
      <div
        class="styles__Container-rvovn3-0 kMwTVK"
      >
        <div
          class="styles__TextContainer-rvovn3-1 bheySt"
        >
          <h2
            class="styles__Title-sc-1b7ft9h-0 cjtgIq"
          >
            Grid Two Column
          </h2>
          <div
            class="styles__Container-sc-1ba2h2-0 cPpcCF"
          >
            Lorem ipsum, dolor sit amet consectetur adipisicing elit.
        Voluptatum aut, eaque velit ullam exercitationem architecto
        excepturi asperiores quos cupiditate, explicabo, minima impedit
        distinctio? Officia dicta, possimus modi recusandae explicabo magni!
          </div>
        </div>
        <div
          class="styles__ImageContainer-rvovn3-2"
        >
          <img
            alt="Grid Two Column"
            class="styles__Image-rvovn3-3"
            src="assets/images/javascript.svg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<GridTwoColumn /> should render with two column grid not background 1`] = `
<div>
  <div
    class="styles__Container-d23vz7-0 bMuKEd"
    id=""
  >
    <div
      class="styles__Container-sc-1abwwhx-0 kXNRnk"
    >
      <div
        class="styles__Container-rvovn3-0 kMwTVK"
      >
        <div
          class="styles__TextContainer-rvovn3-1 bheySt"
        >
          <h2
            class="styles__Title-sc-1b7ft9h-0 cjtgIq"
          >
            Grid Two Column
          </h2>
          <div
            class="styles__Container-sc-1ba2h2-0 cPpcCF"
          >
            Lorem ipsum, dolor sit amet consectetur adipisicing elit.
        Voluptatum aut, eaque velit ullam exercitationem architecto
        excepturi asperiores quos cupiditate, explicabo, minima impedit
        distinctio? Officia dicta, possimus modi recusandae explicabo magni!
          </div>
        </div>
        <div
          class="styles__ImageContainer-rvovn3-2"
        >
          <img
            alt="Grid Two Column"
            class="styles__Image-rvovn3-3"
            src="assets/images/javascript.svg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
